experimental/backwards-compatability/node10/package.json

```diff
@@ -17,5 +17,8 @@
     "typescript": "4.4.4"
   },
   "author": "OpenTelemetry Authors",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
```

Comment:
I think this makes not that much sense in a nodejs 10 backward compat test :o). It's time to remove them or rename/add one for 14 and 16